acceptance-tests/src/main/java/io/blueocean/ath/pages/blue/GithubAddServerDialogPage.java

```diff
@@ -3,17 +3,18 @@
 import io.blueocean.ath.WebDriverMixin;
 import io.blueocean.ath.CustomExpectedConditions;
 import io.blueocean.ath.WaitUtil;
-import org.apache.log4j.Logger;
 import org.openqa.selenium.By;
 import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.slf4j.LoggerFactory;
+import org.slf4j.Logger;
 
 import javax.inject.Inject;
 import javax.inject.Singleton;
 import java.util.regex.Pattern;
 
 @Singleton
 public class GithubAddServerDialogPage implements WebDriverMixin {
-    private Logger logger = Logger.getLogger(GithubAddServerDialogPage.class);
+    private Logger logger = LoggerFactory.getLogger( GithubAddServerDialogPage.class);
```

Comment:
This change appears unrelated to guava. Do we need to do it in this PR?
```suggestion
    private Logger logger = LoggerFactory.getLogger(GithubAddServerDialogPage.class);
```